src/routes/tsoa/dbconnection.ts

```
@@ -77,11 +77,16 @@ export class DbConnectionController extends Controller {
       });
     }
 
+    // This seems like an existing issue... if the user exists we should just return this...
+    // BUT we are leaking the existence of a user...
+    // so we could always return `ok` but DO NOT actually change the password
+    // TESTED - and it works, but I think it's because we're making multiple records... but always picking the first
     // Store the password
     await ctx.env.data.passwords.create(client.tenant_id, {
       user_id: user.id,
       password: body.password,
     });
+    // no, user_id is a primary key... so how can this be created again? Does it silently fail?
```

:exploding_head:   I don't see how this works if the user exists... are we swallowing an error?

I just want to be sure that we're not relying on a `.takeFirst()` SQL statement where the order of records in the table is what stops us getting hacked :sunglasses:

---

This test is running against the in-memory adapter right? Think it might fail against a sql adapter?

---

ahhhhhh :facepalm: 

I was confusing myself because of course I have been working with SQLite :facepalm: 

Sounds like I should copy over a lot of these tests to the hono/testing+SQLite tests :sunglasses: